iphone/Classes/MediaModule.m

```diff
@@ -1109,7 +1109,7 @@ - (void)openPhotoGallery:(id)args
 
   NSArray *types = (NSArray *)[args objectForKey:@"mediaTypes"];
 #if IS_SDK_IOS_14
-  if ([TiUtils isIOSVersionOrGreater:@"14.0"] && ([args objectForKey:@"selectionLimit"] || [TiUtils boolValue:[args objectForKey:@"allowMultiple"] def:NO])) {
+  if ([TiUtils isIOSVersionOrGreater:@"14.0"] && [TiUtils boolValue:[args objectForKey:@"allowMultiple"] def:NO]) {
```

Comment:
Any reason why not to use the PHPicker by default on iOS 14 all the time? I'd love to use it from now on - the UX is amazing, also for single selections!

Reply:
1. In new PHPickerViewController, there is no option for image editing. I think this is important feature and we can not remove it in minor release. (I'll create a ticket to deprecate editing property, in case it is not supported by Apple in future releases).
2. In PHPickerViewController, video file need to copy locally as per discussion https://developer.apple.com/forums/thread/652695. I think this is not the best way. May be its handling can be changed by Apple. 
So I have limited this API to multiple selection only. It will be updated in future release.

In case you want to use the API with single selection, set  `Ti.Media.allowMultiple = true` and Ti.Media.selectionLimit = 1. But yes, a minor change required in your callback.

Reply:
Ohh, thats sad! Didn't know they missed parity when updating the API. Alrighty, thanks for the clarification Vijay! 🙂